### PR TITLE
Helper tool now accepts package id and version urlencoded to allow passing weird unicode stuff

### PR DESCRIPTION
**Pull request by agr**
Fix for the [eng-491](https://github.com/NuGet/Engineering/issues/491).

There are issues with passing unicode characters through a command line, this change introduces the ability for the helper to to URLDecode for the PackageId and PackageVersion parameters so user would be able to proceed with the failed package.

### CODE REVIEW

File: src/Validation.Helper/MarkClean.cs [hunk 1]
```diff
@@ -43,7 +43,9 @@ string galleryBaseAddress
             _containerName = containerName;
 
             _packageId = JobConfigurationManager.GetArgument(jobArgsDictionary, CommandLineArguments.PackageId);
+            _packageId = System.Web.HttpUtility.UrlDecode(_packageId);
```
Comment (loic-sharma) [inline src/Validation.Helper/MarkClean.cs]: Add a using System.Web?

Comment (joelverhagen) [inline src/Validation.Helper/MarkClean.cs]: Unit tests?

Comment (agr) [inline src/Validation.Helper/MarkClean.cs]: I just didn't want to add a `using` for the two lines of code. Added

Comment (agr) [inline src/Validation.Helper/MarkClean.cs]: Added

File: tests/Validation.Helper.Tests/MarkCleanFacts.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.Extensions.Logging;
```
Comment (joelverhagen) [inline tests/Validation.Helper.Tests/MarkCleanFacts.cs]: copyright header

File: tests/Validation.Helper.Tests/MarkCleanFacts.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+using Moq;
+using System;
```
Comment (joelverhagen) [inline tests/Validation.Helper.Tests/MarkCleanFacts.cs]: `System.*` on top

File: tests/Validation.Helper.Tests/Validation.Helper.Tests.nuget.props [hunk 4]
```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
```
Comment (joelverhagen) [inline tests/Validation.Helper.Tests/Validation.Helper.Tests.nuget.props]: delete

**Pull request merged**